text/0265-event-vision.md

```diff
@@ -0,0 +1,65 @@
+# Event Basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to document and agree on a few basics.
+
+### What are OpenTelemetry Events?
+
+OpenTelemetry Events are a type of OpenTelemetry Log that requires an event name and follows a specific structure implied by that event name.
+
+They are a core concept in OpenTelemetry Semantic Conventions.
+
+### OTLP
+
+Since OpenTelemetry Events are a type of OpenTelemetry Log, they share the same OTLP log data structure and pipeline.
+
+### API
+
+OpenTelemetry should provide a (user-facing) Log API that includes the capability to emit OpenTelemetry Events.
```

Comment:
Apart from mentioning `Log` way more than I think this covers the "Event" portion of the discussion (whether or not I fully agree on it being combined 🥲 ).

However, I would "suggest" that the section should mention (or reference a new Issue perhaps) that as part of the (user-facing) Log API there is discussions required about set of capabilities that it will have.

Or maybe just say

OpenTelemetry should provide a (user-facing) Log API that includes the capability to emit OpenTelemetry Events `to enforce the basic general semantic conventions.`

"enforce" might be a bit harsh, so some other wording about "validating" or maybe just to provide context that, "ensuring that they are followed" etc.

Reply:
> However, I would "suggest" that the section should mention (or reference a new Issue perhaps) that as part of the (user-facing) Log API there is discussions required about set of capabilities that it will have.

added this to the open questions section